crud/select/plan.lua

```diff
@@ -182,6 +182,12 @@ function select_plan.new(space, conditions, opts)
         return nil, err
     end
 
+    if scan_after_tuple ~= nil then
+        if scan_iter == box.index.REQ or scan_iter == box.index.EQ then
+            table.insert(conditions, conditions[scan_condition_num])
+        end
+    end
```

Comment:
Patch is more laconic and it's great but this change is not trivial. Please put a comment here.